Makefile

```
@@ -1,26 +1,30 @@
-BUILD_AMD64 ?= 1
-BUILD_ARM32 ?= 1
-BUILD_ARM64 ?= 1
+BUILDX_ARGS =
+EXTRA_CARGO_ARGS =
+
+PUSH ?= 
+LOAD ?=
+LOCAL_ARCH = $(shell uname -m)
+ifeq ($(LOAD), 1)
+PLATFORMS ?= $(LOCAL_ARCH)
+ifneq (1, $(words $(PLATFORMS)))
+$(error Cannot load for more than one platform: [$(PLATFORMS)])
+endif
+else
+PLATFORMS ?= amd64 arm64 arm/v7
+endif
+
+null  :=
+space := $(null) #

```

is there. reason for the `#`
```suggestion
space := $(null)
```

---

`null` is an empty variable, so `space := $(null)` expands to an empty variable as well, adding a trailing space won't work as it would get trimmed, Adding a comment `#` after it somehow makes that space to be registered as part of the value, so this is needed.

---

im surprised this doesn't comment follow up characters where we are using `$space` concatenated with other strings but fine to keep it if workflows are passing